plugin/normalize_test.go

```diff
@@ -138,3 +140,23 @@ func TestHostNormalizeExact(t *testing.T) {
 		}
 	}
 }
+func TestIsSubDomain(t *testing.T) {
```

Comment:
this should have been rather Benchmark test https://pkg.go.dev/testing#hdr-Benchmarks